tests/e2e/e2e_setup_test.go

```diff
@@ -24,7 +24,56 @@ import (
 	"github.com/osmosis-labs/osmosis/v7/tests/e2e/util"
 )
 
-const maxRetries = 10 // max retries for json unmarshalling
+var (
+	// common
+	maxRetries             = 10 // max retries for json unmarshalling
+	validatorConfigsChainA = []*chain.ValidatorConfig{
+		{
+			Pruning:            "default", // default, nothing, everything, or custom
```

Comment:
let's move the comments to the struct definition in chain package instead please